resources/user-guides/mine-to-pool.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: static_page
+title: How to mine on a pool
+---
+# Selecting a pool
+
+There are many pools to choose from, a list is available at
+[moneropools.com](https://moneropools.com). Mining on a larger pool could mean
```

Comment:
shouldn't be a problem, developers often redirect to moneropools when somebody ask to suggest a pool